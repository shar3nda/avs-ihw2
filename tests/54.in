fEpy&)R8g 1EsSTAG!Md2B&97RE~Nf4uXMf!q~D2bHhc\7d('A @b(aCm,W}[$kc=g/5rn&[4$kgT