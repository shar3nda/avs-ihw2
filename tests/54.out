Lowercase: 24, uppercase: 20